reverie/backend_server/reverie.py

```diff
@@ -398,6 +398,7 @@ def start_server(self, int_counter):
           #  "persona": {"Klaus Mueller": {"movement": [38, 12]}}, 
           #  "meta": {curr_time: <datetime>}}
           curr_move_file = f"{sim_folder}/movement/{self.step}.json"
+          create_folder_if_not_there(curr_move_file)
```

Comment:
this should be merged, no?

```
movementFolder = f"{sim_folder}/movement"
if not os.path.exists(movementFolder):
     os.mkdir(movementFolder)
```

seems to be a bug. 

i also hit it, as noted by other [here](https://github.com/joonspk-research/generative_agents/issues/40).